.circleci/config.yml

```diff
@@ -134,6 +134,24 @@ jobs:
       - run: operator-courier verify ./deploy/olm/kubernetes
       - run: operator-courier verify ./deploy/olm/openshift
 
+  deploy-release-artefacts-github:
+    docker:
+      - image: cibuilds/github
+    steps:
+      - checkout
+      - run: |
+          template_image="dynatrace-oneagent-operator:snapshot"
+          current_image="dynatrace-agent-operator:${CIRCLE_TAG}"
+          mkdir artefacts
+          sed "s/${template_image}/${current_image}/g" deploy/kubernetes.yaml > artefacts/kubernetes.yaml
+          sed "s/${template_image}/${current_image}/g" deploy/kubernetes.yaml > artefacts/openshift.yaml
```

Comment:
Use openshift.yaml as base file, plus the one on master uses Quay whereas the releases are on RHCC, so the image registry needs to be replaced as well.

Reply:
fixed